tests/product/test_connectors.py

```diff
@@ -297,7 +300,7 @@ def test_connector_remove(self):
         for host in self.cluster.all_hosts():
             self.assert_has_default_connector(host)
 
-        missing_connector_message = """[Errno 1] 
+        missing_connector_message = """[Errno 1]
```

Comment:
Did you intend to remove this single space character? I think this test does a string match and will fail if the error doesn't exactly correspond to what is expected.